backend/routes/admin/admin_router.py

```
@@ -1,287 +1,179 @@
         return ErrorResponseModel(
-            status="error", message=f"Failed to delete team: {str(e)}"
+            status="error", message=f"Failed to delete institution: {str(e)}"
         )
 
 
-@admin_router.get("/get-all-teams", response_model=ResponseModel)
+@admin_router.get("/get-all-institutions", response_model=ResponseModel)
 @verify_admin_role
-async def get_teams_endpoint(
+async def get_institutions_endpoint(
     session: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)